fetch.bs

```diff
@@ -2049,6 +2051,29 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<h3 id=http-cache-partition>HTTP Cache partition</h3>
+
+<p>To determine the associated <dfn export id=http-cache>HTTP cache</dfn> partition, given
+<var>request</var>, run these steps:
+
+<ol>
+ <li>
+  <p>If <var>request</var>'s <a for=request>client</a> is null, terminate these steps.</li>
+
+ <li>
+  <p>Let <var>environmentSettings</var> be <var>request</var>'s <a for=request>client</a>.</li>
+
+ <li>
+  <p>Let <var>topLevelOrigin</var> be <var>environmentSettings</var>'s top-level origin and
+  <var>origin</var> be <var>environmentSettings</var>'s origin.</li>
```

Comment:
This should be `<a for="environment settings object">origin</a>`.

Reply:
done